### PR TITLE
MIG-3897 and MIG 3874

### PR DESCRIPTION
**Pull request by moiznalwalla**
https://enterprisedb.atlassian.net/browse/MIG-3897
https://enterprisedb.atlassian.net/browse/MIG-3874

### CODE REVIEW

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx [hunk 1]
```diff
@@ -113,4 +113,8 @@ See an important [note](../01_mp_schema_extraction/#about-file-encoding) about t
    ```shell
     impdp <Username>@<ConnectIdentifier> DIRECTORY=DMPDIR TRANSFORM=OID:n,SEGMENT_ATTRIBUTES:n SQLFILE=YourSchemas.sql DUMPFILE=schemas_metadata.dump parfile=import.par
    ```
-See an important [note](../01_mp_schema_extraction/#about-file-encoding) about the file encoding format expected by the Migration Portal.  
+See an important [note](../01_mp_schema_extraction/#about-file-encoding) about the file encoding format expected by the Migration Portal.
+
+## Known Issues and Additional Notes
```
Comment (drothery-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx:118]:
```suggestion
## Known issues and additional notes
```

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx [hunk 2]
```diff
@@ -113,4 +113,8 @@ See an important [note](../01_mp_schema_extraction/#about-file-encoding) about t
...
+
+- Migration Portal might fail to parse your SQL file if you create a database link using the IDENTIFIED BY clause with Oracle's quote operator, for example, `IDENTIFIED BY VALUES q'[:1]'`. To parse your file successfully, try using an actual password, for example, `IDENTIFIED BY my_password`.
```
Comment (drothery-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx:120]:
```suggestion
- Migration Portal might fail to parse your SQL file if you create a database link using the IDENTIFIED BY clause with Oracle's quote operator; for example, `IDENTIFIED BY VALUES q'[:1]'`. To parse your file successfully, try using an actual password; for example, `IDENTIFIED BY my_password`.
```
I'm pretty sure that we want to use semicolons but you might want to confirm that with Betsy.

Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx:120]: Fixed.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx:118]: Fixed.

**Comment by drothery-edb** [comment]
We might want to create a new topic titled Known issues and additional notes under Performing a schema extraction and move the content in the following sections there:
- https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/#additional-notes
- https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/#about-file-encoding
- https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility/#known-issues-and-additional-notes

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx [hunk 3]
```diff
@@ -163,7 +163,15 @@ Exclude these Oracle systems schemas while generating the SQL dump file.
 -   Directories
 -   Users
 -   RLS policy
--   Queues
+-   Queues *
+-   Library
+-   Alter Function
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx:168]: I don't think we should list Alter Function, Alter Package, and Alter Type here like this. They are not object types. Rather, we should probably have a note that indicates that with the exception of ALTER TABLE statements that add constraints, Migration Portal does not process ALTER statements that may added to the DDL files generated by the Oracle data pump utilities.
@umangjtank can you confirm that no other ALTER statements other than ALTER TABLE statements that create constraints are processed by Migration Portal?

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
See comments about list of unsupported types.

**Comment by moiznalwalla** [comment]
> We might want to create a new topic titled Known issues and additional notes under Performing a schema extraction and move the content in the following sections there:
> 
> * https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/#additional-notes
> * https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/#about-file-encoding
> * https://deploy-preview-3009--edb-docs-staging.netlify.app/docs/migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility/#known-issues-and-additional-notes

@mlewandowski-edb I took a shot at implementing this let me know if it looks OK?

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Known issues and additional notes"
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:2]: Some of the items in this section are limitations and not necessarily known issues.  I suggest changing the title of the section to "Known issues, limitations, and additional notes" or "Known isues, limitations, and notes".

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+---
+
+This page lists known issues and additional notes for:
+
+ - [Migration Portal](#migration-portal)  
+ - [EDB DDL Extractor](#edb-ddl-extractor) 
+ - [Data pump utility](#data-pump-utility)
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:10]: Data pump is not a single utility.  It Data Pump is an oracle feature that provides a set of utilities.  When we reference it, we probably should say "Oracle Data Pump utilities" (note capitalization of "Data" and "Pump").

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Known issues and additional notes"
+
+---
+
+This page lists known issues and additional notes for:
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:6]: Change "known issues and additional notes" to "known issues, limitations, and notes".

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+-   The EDB DDL Extractor extracts `nologging` tables as normal tables. Once these tables are migrated to EDB Postgres Advanced Server, WAL log files are created.
+-   The EDB DDL Extractor extracts objects only with VALID status. For any objects that have INVALID status that you want Migration Portal to assess, first update them to VALID.
+-   The EDB DDL Extractor doesn't extract objects that were obfuscated using the Oracle wrap feature. As such, these objects aren't included in the set of DDL assessed by the Migration Portal. If you want to assess these objects and migrate them to EDB Postgres Advanced Server, replace the wrapped versions of the objects with nonwrapped versions. See [Wrapped objects](#wrapped-objects) for more information.
+-   The EDB DDL Extractor creates Global Temporary tables to store the schema names and their dependency information. These tables are dropped at the end of successful extraction.
+-   The EDB DDL Extractor script doesn't extract schemas whose name starts with `PG_` because PostgreSQL doesn't support it. If you want to extract these schemas, you must change the name of the schema before extraction.
+
+## Data pump utility
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:40]: As noted in earlier comment, this sub-section title should probably be changed to "Oracle Data Pump utilities".

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
See comments on changes.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx (hunk 4 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:2]: Renamed to Known issues, limitations, and notes

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx (hunk 5 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:10]: Done.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx (hunk 6 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:6]: Done.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx (hunk 7 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx:40]: Done.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx (hunk 3 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/index.mdx]: Added the following in the known issues, limitations, and notes page.

under Migration Portal:
"Except for ALTER TABLE and ALTER TRIGGER, Migration Portal does not process any other ALTER statement in the DDL."

under Oracle Data Pump utilities:
"The DDL generated by Oracle Data Pump utilities might contain ALTER STATEMENTS such as ALTER FUNCTION, ALTER PACKAGE, and ALTER TYPE. Migration Portal does not process these statements."

**Review by mlewandowski-edb — APPROVED** [review]
The changes look good.

**Pull request merged**